src/PresentationalComponents/Breadcrumbs/ConnectedBreadcrumbs.js

```diff
@@ -7,6 +7,7 @@ class ConnectedBreadcrumbs extends Component {
         super(props);
         this.onNavigate = this.onNavigate.bind(this);
         this.calculateBreadcrumbs = this.calculateBreadcrumbs.bind(this);
+        console.warn('This component will be removed in future release, do not use it anymore!');
```

Comment:
Can we come up with a date for these?

Reply:
Sure I don't really thing anybody is using it anyways.